src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementServiceTest.kt

```diff
@@ -2,44 +2,65 @@ package uk.gov.justice.digital.hmpps.welcometoprison.model
 
 import io.mockk.every
 import io.mockk.mockk
+import io.mockk.verify
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.welcometoprison.model.MoveType.PRISON_RECALL
+import uk.gov.justice.digital.hmpps.welcometoprison.model.MoveType.PRISON_TRANSFER
 import uk.gov.justice.digital.hmpps.welcometoprison.model.basm.BasmService
 import uk.gov.justice.digital.hmpps.welcometoprison.model.prison.PrisonService
+import uk.gov.justice.digital.hmpps.welcometoprison.model.prisonersearch.PrisonerSearchService
+import uk.gov.justice.digital.hmpps.welcometoprison.model.prisonersearch.response.MatchPrisonerResponse
 import java.time.LocalDate
 
 class MovementServiceTest {
   private val prisonService: PrisonService = mockk()
   private val basmService: BasmService = mockk()
-  private val movementService = MovementService(basmService, prisonService)
+  private val prisonerSearchService: PrisonerSearchService = mockk()
+  private val movementService = MovementService(basmService, prisonService, prisonerSearchService)
 
-  @Test
-  fun `happy path`() {
+  private fun movementFactory(moveType: MoveType = PRISON_RECALL, prisonNumber: String? = "A1234AA", pncNumber: String? = "99/123456J"): Movement {
+    return Movement(
+      id = "1",
+      firstName = "JIM",
+      lastName = "SMITH",
+      dateOfBirth = LocalDate.of(1991, 7, 31),
+      prisonNumber,
+      pncNumber,
+      date = LocalDate.of(2021, 1, 2),
+      fromLocation = "MDI",
+      moveType
+    )
+  }
```

Comment:
There's a lot of repetitive code constructing `Movement` objects that are very similar.
This is a factory method which helps, but it isn't used in the test below.
Consider making copies of a prototype `Movement` object, modifying just those fields that differ from the prototype.
This might be more effective than the factory method - since `Movement.copy` is a more general factory method.